### DISCUSSION TITLE
Entware installation says "opkg: not found"

### DISCUSSION THREAD

**Discussion by TrueBlueRoo** [Guilouz/Creality-Helper-Script-Wiki#588] [Issues]
After installing Entware it comes up with _opkg: not found_
There is a file opkg in /usr/data/opt/bin but there seems a link missing to call it from anywhere.

**Comment by Guilouz** [comment]
Did you see what is reported at the end of the Entware installation?

Disconnect and reconnect SSH session, and you can now install packages with: opkg install <packagename>

**Reply by TrueBlueRoo** [reply]
It was fine with the installation. No errors whatsoever.
When I called e.g. opkg install nano it stated opkg is unknown.
I solved it through adding the folder to the PATH:

`export PATH=$PATH:/usr/data/opt/bin`

After that I could call it from everywhere.

**Comment by alvaropsouza** [comment]
That absolutely works for me: https://github.com/Guilouz/Creality-Helper-Script-Wiki/discussions/510#discussioncomment-9079045
